js/plugins/googleai/src/gemini.ts

```diff
@@ -139,7 +144,30 @@ export const gemini15Flash = modelRef({
       systemRole: true,
       output: ['text', 'json'],
     },
-    versions: ['gemini-1.5-flash-001'],
+    versions: [
+      'gemini-1.5-flash',
+      'gemini-1.5-flash-001',
+      'gemini-1.5-flash-002',
+      'gemini-1.5-flash-8b-exp-0924',
+      'gemini-1.5-flash-8b-exp-0827',
+      'gemini-1.5-flash-exp-0827',
```

Comment:
The [docs for experimental models](https://ai.google.dev/gemini-api/docs/models/experimental-models#use-an-experimental-model) state that:

> Support for an experimental model can change at any time.

With that in mind, should these models be marked as [unstable](https://github.com/firebase/genkit/blob/adc8ad81759f49b2791a3a87f7fa2a38d137db24/js/ai/src/model.ts#L152) in Genkit?
We'd separate them from `gemini15Flash`, `gemini15Flash8B` and `gemini15Pro`, which would only contain stable models.

```js
export const gemini15FlashExp = modelRef({
  name: 'googleai/gemini-1.5-flash-exp-0827',
  info: {
    label: 'Google AI - Gemini 1.5 Flash Experimental',
    stage: 'unstable',
    supports: {
      // (...)
    },
    versions: ['gemini-1.5-flash-exp-0827'],
  },
  configSchema: GeminiConfigSchema,
});
```

Same comment for 1.5 Flash 8B (`gemini15Flash8BExp`) and 1.5 Pro (`gemini15ProExp`).

Reply:
The ways the model refs work is that the versions array is only impacting what versions are shown as available in the Developer UI, so changing these out when they lose support isn't a breaking change in the framework.